### PR TITLE
feat(description-list): horizontal variant  - TWIG-39

### PR DESCRIPTION
**Pull request by planctus**
…

# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-description-list/README.md [hunk 1]
```diff
@@ -23,7 +24,8 @@ npm install --save @ecl-twig/ec-component-description-list
 <!-- prettier-ignore -->
 ```twig
 {% include 'path/to/description-list.html.twig' with { 
-  items: [ 
+  variant: horizontal,
```
Comment (papegaill) [inline src/ec/packages/ec-component-description-list/README.md]: maybe you have to surround the value of single quotes

File: src/ec/packages/ec-component-description-list/description-list.html.twig [hunk 2]
```diff
@@ -23,10 +24,15 @@
 
 {% set _items = items|default([]) %}
 {% set _css_class = 'ecl-description-list' %}
+{% set _variant = variant|default('') %}
 {% set _extra_attributes = '' %}
 
 {# Internal logic - Process properties #}
 
+{% if (_variant is not empty) %}
```
Comment (papegaill) [inline src/ec/packages/ec-component-description-list/description-list.html.twig]: parentheses around the condition really needed?

Comment (planctus) [reply src/ec/packages/ec-component-description-list/description-list.html.twig]: not at all.. :)

File: src/ec/packages/ec-component-description-list/__snapshots__/description-list.test.js.snap [hunk 3]
```diff
@@ -47,6 +47,43 @@ exports[`EC - Description list Default renders correctly 1`] = `
 </dl>
 `;
 
+exports[`EC - Description list Default renders correctly in the horizontal variant 1`] = `
+<dl
+  class="ecl-description-list"
```
Comment (emeryro) [inline src/ec/packages/ec-component-description-list/__snapshots__/description-list.test.js.snap]: this is not the horizontal version tested here (missing css class)

File: src/ec/packages/ec-component-description-list/package.json [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
   "author": "European Commission",
   "license": "EUPL-1.1",
-  "version": "2.9.1",
+  "version": "2.11.0",
   "description": "ECL Twig - EC Description list",
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@ecl/ec-component-list": "2.11.0",
```
Comment (emeryro) [inline src/ec/packages/ec-component-description-list/package.json]: it should refers to component `ec-component-description-list` (version 2.11.0).
component `ec-component-list` is deprecated

File: src/ec/packages/ec-component-description-list/__snapshots__/description-list.test.js.snap (hunk 3 above)
Comment (planctus) [reply src/ec/packages/ec-component-description-list/__snapshots__/description-list.test.js.snap]: Oh, true, i assumed that the variant was coming from the spec package without checking it...

**Comment by emeryro** [comment]
Looks good for me.
@papegaill could you validate too, as you requested changes?

**Pull request merged**